packages/drivers/file-driver/src/fileDocumentDeltaConnection.ts

```diff
@@ -130,7 +130,7 @@ export class ReplayFileDeltaConnection
                 maxMessageSize: ReplayMaxMessageSize,
                 summary: {
                     idleTime: 5000,
-                    maxOps: 1000,
+                    maxOps: 100,
```

Comment:
Why change this? If we are ignoring server configurations, this should not matter right?

Reply:
Also, the snapshot tests may break because of this change. Currently, the snapshots files are saved are at 1000 ops interval. The test will now generate summary at 100 ops and it will not find a corresponding snapshot file.

Reply:
reverted

Reply:
Actually, the snapshot tests will break because of the change in container runtime. You might have to pass in the previous configurations when creating the container runtime (if it already doesn't do it).

Reply:
snapshot tests passed! :)

Reply:
My bad. The snapshot tests don't even spawn a summarizer. They do the summaries manually!